tools/bigquery-to-xml/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dicttoxml==1.7.4
```

Comment:
unused?

Reply:
This import actually brings in the appropriate json2xml library which is used; if preferable I could import json2xml directly, but I imagine it would mostly be a change for readability